### DISCUSSION TITLE
开发者认证问题

### DISCUSSION THREAD

**Discussion by jianjianxu** [zepp-health/zeppos-forum#314] [General]
提交认证了好几周了，没任何动静，请教怎么办

官方飞书也停用了，只剩这里了

**Comment by MyoungXUE** [comment]
Hello @jianjianxu , may I ask if the Feishu link on https://docs.zepp.com/zh-cn/docs/guides/community/this document has become invalid? I can still access it normally. The Feishu community has been operating normally. Please try again and contact us in the Feishu community after success. It is recommended to use English for discussion in the Github discussion area, which will help developers from all over the world.

**Reply by jianjianxu** [reply]
`https://docs.zepp.com/zh-cn/docs/guides/community/this` This link is dead.

I also tyied this:
![image](https://github.com/zepp-health/zeppos-forum/assets/4576010/fe41590c-b77c-43d1-887d-ce821ef4fabe)

It's dead too. It shows `Unable to join, the inviter has left the group`.

**Reply by MyoungXUE** [reply]
![飞书20240229-103529](https://github.com/zepp-health/zeppos-forum/assets/107526244/49a6f356-b152-4655-9562-eb103dd72895)


Thank you for your feedback, I know the problem, please scan this QR code to join the group first.
